config/environments/production.rb

```diff
@@ -56,6 +56,13 @@
   # Send deprecation notices to registered listeners.
   config.active_support.deprecation = :notify
 
+  # Configure Cache-Control header used for serving static files by ActionDispatch::Static middleware
+  config.static_cache_control = "public, max-age=7200"
```

Comment:
Is this actually necessary given static assets are served by nginx in production?